AdvLoggerPkg/AdvLoggerPkg.dec

```diff
@@ -81,6 +81,10 @@
   #
   gAdvLoggerPkgTokenSpaceGuid.PcdAdvancedFileLoggerForceEnable|TRUE|BOOLEAN|0x00010184
 
+  ## PcdAdvancedLoggerAutoWrapEnable - Automatically wrap around the LogCurrent cursor when it reaches the end of the log

```

Comment:
We should mention here that this only occurs at runtime (after exit boot service even is triggered)

Reply:
Updated.